### PR TITLE
mobile version portfolio using html and css

### PR DESCRIPTION
**Pull request by osmanwako**
## mobile version portfolio using html and CSS
- workout section , introduction and about my self using grid box
-  flex box used in all internal container of workout section , introduction and about-mysef section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+            </li>
+        </ul>
+            </span>
+        </section>
+
+         <section class="grid-item grid-item-3 work-section">
+            <div id="work-container">
```
Comment (victorokoroji) [inline index.html:50]: - [ ] KIndly ensure you add the hover interactions on the buttons as shown on the figma design which helps to increase user experience 👍🏽 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                <!-- -------------------1 -->
+                <span class="work-content">
+                    <div class="img-content">
+                      <img src="./assets/images/work-section/project2.png" alt="work 1" class="" />
+                    </div>
+                      <h1 class="work-title">Multi-Post<br>
+                        Stories</h1>
+                      <div class="work-role">
+                        <h3 class="work-role-list role-job ">CANOPY</h3>
+                        <div class="work-role-list  role-dot">&#8901;</div>
```
Comment (victorokoroji) [inline index.html]: - [ ] KIndly remove all the visible black dots from all the project cards to match the figma design. YOu can actually create the dot effect by styling an empty div as you have done.

your design 👇🏽 
image
 
figma design 👇🏽 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
+    <span class="p-more-information">
+        <a class="moresee-link">Get my resume</a>
+        </span>
+        <p class="planguage">Languages</p>
```
Comment (victorokoroji) [inline index.html]: - [ ] Kindly ensure that your list of technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li tag which will make your website more semantic which in turn will increase your website accessibility 💯

Comment (victorokoroji) [inline index.html]: - [ ] An arrow is missing at the side of your language header. KIndly add it 👍🏽 

figma design 👇🏽 
image

your design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @osmanwako, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+            </li>
+        </ul>
+            </span>
+        </section>
+
+         <section class="grid-item grid-item-3 work-section">
+            <div id="work-container">
```
Comment (adel-gu) [inline index.html:50]: - [ ] As requested by the previous reviewer ensure adding interactions to all the existing buttons ✌

image

- Here is a great article about [CSS basic 8 - :hover, :active, :focus](https://dev.to/daaahailey/css-basic-8-hover-active-focus-4d78)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+             I’d love to help with it 
+        </p>
+        <input class="input-bg aboutmyself-input" type="text" placeholder="Osman">
+        <input type="email" class="input-bg aboutmyself-input" placeholder="osman@gmail">
+        <textarea class="input-bg aboutmyself-message" cols="100" placeholder="Describe yourself here..."></textarea>
+        <input type="submit" class="input-bg aboutmyself-getintouch" value="Get in touch">
+      </span>
+    </div>
+</section>
+
```
Comment (adel-gu) [inline index.html:223]: ### [OPTIONAL] The comments below are CSS-related changes.

- [ ] Please be aware that the mobile screen size is a range of **min 375px - 786px** currently your website looks amazingly responsive on screen resolution less than **420px** 👌, Pease focus on the responsiveness on the resolution **min 375px - 786px** instead.

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+            <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
+    <span class="p-more-information ">
+        <a class="moresee-link btn-action-event">Get my resume</a>
+        </span>
+        <ul class="expand-ul"><li><a class="expand-ula">Languages</a></li><li><input type="image" src="./assets/images/arrowdown.png" class="bn-img-down"></li></ul>
```
Comment (adel-gu) [inline index.html]: - Well done adding the icon that the reviewer asked for 👏However the look is still different from what the Figma design is displaying 

### Your Design 
image

### Figma Design
image

- [ ] Please ensure to minimize the button arrow down icon to look like the Figma design as much as possible 💪

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️ 

**Hi @osmanwako 👋🏼** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- GitHub Flow correctly used ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @osmanwako, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions
- I think you misunderstood the recommendation of the previous reviewer. He meant to say that your button should look like these:
image
While you button in normal state looks like this:
image
While on click button looks like this:
image
You were also guided to read about this here:
Here is a great article about [CSS basic 8 - :hover, :active, :focus](https://dev.to/daaahailey/css-basic-8-hover-active-focus-4d78)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**